target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/common.c

```diff
@@ -1546,8 +1510,8 @@ static int __init rtl83xx_sw_probe(struct platform_device *pdev)
 		return err;
 	}
 
-	priv->msts = devm_kcalloc(priv->dev,
-				  priv->n_mst - 1, sizeof(struct rtldsa_mst),
+	priv->msts = devm_kzalloc(priv->dev,
```

Comment:
`kcalloc` is also zeroing out everything - but it meant to allocate an array. So please don't change from `kcalloc` to `kzalloc`